modules/ipi-install-replacing-a-bare-metal-control-plane-node.adoc

```diff
@@ -0,0 +1,182 @@
+    machine.openshift.io/cluster-api-machine-type: master
+  name: kni1-master-2
+  namespace: openshift-machine-api
+spec:
+  metadata: {}
+  providerSpec:
+    value:
+      apiVersion: baremetal.cluster.k8s.io/v1alpha1
+      customDeploy:
+        method: install_coreos
+      hostSelector: {}
+      image:
+        checksum: ""
+        url: ""
+      kind: BareMetalMachineProviderSpec
+      metadata:
+        creationTimestamp: null
+      userData:
+        name: master-user-data-managed
+EOF
+----
++
+. To define and create the `BareMetalHost`, `Secret`, and `MachineSet` objects in a single YAML file, run the following command:
++
+[source,terminal]
+----
+$ oc create -f example.yaml
+----
++
+The provisioning process uses the baremetal-operator to install RHCOS and prepare the host to be added to the cluster. 
++
+. To view the `baremetalhost` objects, run the following command:
++
+[source,terminal]
+----
+$ oc get bmh -A
+----
++
+.Example output
+[source,terminal]
+----
+NAMESPACE               NAME           STATUS PROVISIONING STATUS     CONSUMER        BMC                     HARDWARE PROFILE   ONLINE   ERROR
+openshift-machine-api   kni1-master-0  OK     provisioned             kni1-worker-0   ipmi://10.19.136.22     dell-raid          true
+openshift-machine-api   master-1       OK     externally provisioned  kni1-master-1   ipmi://172.22.0.1:6231                     true
+openshift-machine-api   master-2       OK     externally provisioned  kni1-master-2   ipmi://172.22.0.1:6232                     true
+openshift-machine-api   master-3       OK     provisioning            kni1-master-3   ipmi://172.22.0.1:6233  unknown            true
+openshift-machine-api   worker-0       OK     provisioned             kni1-worker-1   ipmi://10.19.143.29     dell-raid          true
+----
++
+. Verify that the `BareMetalHost` is added to the cluster:
++
+[source,terminal]
+----
+$ oc get bmh -A
+----
++
+.Example output
+[source,terminal]
+----
+NAMESPACE             NAME     STATUS PROVISIONING STATUS     CONSUMER          BMC                    HARDWARE PROFILE ONLINE ERROR
+openshift-machine-api master-1 OK     externally provisioned  kni1-master-1     ipmi://172.22.0.1:6231                  true 
+openshift-machine-api master-2 OK     externally provisioned  kni1-master-2     ipmi://172.22.0.1:6232                  true
+openshift-machine-api master-3 OK     provisioned             kni1-master-3     ipmi://172.22.0.1:6233 unknown          true
+openshift-machine-api worker-0 OK     provisioned             kni1-worker-0     ipmi://10.19.143.29    dell-raid        true
+----
++
+. Confirm that the node has been added to the cluster:
++
+[source,terminal]
+----
+$ oc get nodes
+----
++
+.Example output
+[source,terminal]
+----
+NAME             STATUS      ROLES    AGE   VERSION
+kni1-master-1    available	 master   141m  v1.18.2
+kni1-master-2    available	 master   141m  v1.18.2
+kni1-master-3    available	 master   4m2s  v1.18.2
```

Comment:
```suggestion
kni1-master-0    available	 master   4m2s  v1.18.2
```

Reply:
And move it to the top of this output because it is sorted

Reply:
Updated as suggested.